Tests/UnitTests/TestHelpers/DiagnosticsEvent+Equatable.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DiagnosticsEvent+Equatable.swift
+//
+//  Created by Antonio Pallares on 28/2/25.
+
+import Foundation
+@testable import RevenueCat
+
+extension DiagnosticsEvent: @retroactive Equatable {
+
+    public static func == (lhs: DiagnosticsEvent, rhs: DiagnosticsEvent) -> Bool {
+        return lhs.version == rhs.version &&
+               lhs.eventType == rhs.eventType &&
+               lhs.properties == rhs.properties &&
+               lhs.timestamp == rhs.timestamp
+    }
+
+}
+
+extension AnyEncodable: @retroactive Equatable {
```

Comment:
Hmm I wonder if this belongs in its own file... I think that's used in other places other than diagnostics?